### PR TITLE
removing jQuery dependency on code and use vanilla js.

### PR DESCRIPTION
**Pull request by iwaduarte**
Basing on the issue created 15 hours ago:

https://github.com/ubports/ubports-installer/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22

 I am doing my first contribution to the open source community. I hope it serves its purpose.
I refactored the code and although I spotted some minors bugs I left untouched as the issue requested it was to remove the jQuery lib from the code and I have done it following the previous coding style (structure). Please let me know if any doubt arises.

### CODE REVIEW

File: src/html/scripts/root.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-require("jquery");
-window.$ = window.jQuery = require("../../node_modules/jquery/dist/jquery.js");
+//require("jquery");
+//window.$ = window.jQuery = require("../../node_modules/jquery/dist/jquery.js");
```
Comment (NeoTheThird) [inline src/html/scripts/root.js]:
```suggestion
```

File: src/html/scripts/ui.js [hunk 2]
```diff
@@ -1,167 +1,215 @@
 const switchHide = (from, to) => {
-  $("." + from).hide();
-  $("." + to).show();
+    const elementsToHide = document.getElementsByClassName(from);
+    elementsToHide.forEach(el => el.display = "none");
+    const elementsToDisplay = document.getElementsByClassName(to);
+    elementsToDisplay.forEach(el => el.style.display = "block");
+
```
Comment (NeoTheThird) [inline src/html/scripts/ui.js]:
```suggestion
```

**Comment by NeoTheThird** [comment]
The linter will also insist on two spaces of indentation, as this is the convention for this project. `npm run lint-fix` will take care of that for you.

**Comment by iwaduarte** [comment]
No worries. Thanks for having my first pull request. If you need more help just let me know.
I have run the lint-fix directive.

**Comment by NeoTheThird** [comment]
So, congrats on getting your first PR merged in six hours :)

If you're interested to help out with some more js, feel free to check out the remaining [good first issues](https://github.com/ubports/ubports-installer/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) here - or take on any of the issues you saw that you mentioned above. The Installer code still kind of suffers from cowboy coding done in the beginning of the project, so it's great to have new eyes to help smooth things out! We also have [some other js projects](https://github.com/ubports?q=&type=&language=javascript), mostly dependencies for the installer, if you're interested in one of those.

**Comment by iwaduarte** [comment]
> So, congrats on getting your first PR merged in six hours :)
> 
> If you're interested to help out with some more js, feel free to check out the remaining [good first issues](https://github.com/ubports/ubports-installer/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) here - or take on any of the issues you saw that you mentioned above. The Installer code still kind of suffers from cowboy coding done in the beginning of the project, so it's great to have new eyes to help smooth things out! We also have [some other js projects](https://github.com/ubports?q=&type=&language=javascript), mostly dependencies for the installer, if you're interested in one of those.

Awesome. I will.

**Comment by NeoTheThird** [comment]
Very sorry, i had to revert these for now, as they introduced some critical issues. They can be revisited later. Thank you for your work, anyways!

**Pull request merged**